xml/System.Threading.Tasks/Task`1.xml

```diff
@@ -2151,9 +2151,9 @@ For operations that do not return a value, you use the <xref:System.Threading.Ta
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" Index="0" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="cancellationToken">The <see cref="T:System.Threading.CancellationToken" /> to monitor for a cancellation request.</param>
+        <summary>Gets a <see cref="T:System.Threading.Tasks.Task`1" /> that will complete when this <see cref="T:System.Threading.Tasks.Task`1" /> completes or when the specified <see cref="T:System.Threading.CancellationToken" /> has cancellation requested.</summary>
+        <returns>The <see cref="T:System.Threading.Tasks.Task`1" /> representing the asynchronous wait.  It may or may not be the same instance as the current instance.</returns>
```

Comment:
```suggestion
        <returns>The <see cref="T:System.Threading.Tasks.Task`1" /> representing the asynchronous wait. It may or may not be the same instance as the current instance.</returns>
```